Client/Frontend/Home/RemoteTabsPanel.swift

```diff
@@ -518,3 +563,82 @@ class RemoteTabsNotLoggedInCell: UITableViewCell {
         super.updateConstraints()
     }
 }
+
+private class HistoryBackButton : UIView {
+    lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = UIConstants.HighlightBlue
+        label.text = Strings.HistoryBackButtonTitle
+        return label
+    }()
+
+    lazy var chevron: ChevronView = {
+        let chevron = ChevronView(direction: .Left)
+        chevron.tintColor = UIConstants.HighlightBlue
+        chevron.lineWidth = RemoteTabsPanelUX.HistoryTableViewHeaderChevronLineWidth
+        return chevron
+    }()
+
+    lazy var topBorder: UIView = self.createBorderView()
+    lazy var bottomBorder: UIView = self.createBorderView()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        userInteractionEnabled = true
+
+        addSubview(topBorder)
+        addSubview(chevron)
+        addSubview(titleLabel)
+
+        chevron.snp_makeConstraints { make in
+            make.left.equalTo(self).offset(RemoteTabsPanelUX.HistoryTableViewHeaderChevronInset)
+            make.centerY.equalTo(self)
+            make.size.equalTo(RemoteTabsPanelUX.HistoryTableViewHeaderChevronSize)
+        }
+
+        titleLabel.snp_makeConstraints { make in
+            make.left.equalTo(chevron.snp_right).offset(RemoteTabsPanelUX.HistoryTableViewHeaderChevronInset)
+            make.right.greaterThanOrEqualTo(self).offset(-RemoteTabsPanelUX.HistoryTableViewHeaderChevronInset)
+            make.centerY.equalTo(self)
+        }
+
+        topBorder.snp_makeConstraints { make in
+            make.left.right.equalTo(self)
+            make.top.equalTo(self).offset(-0.5)
+            make.height.equalTo(0.5)
+        }
+    }
+
+    private func createBorderView() -> UIView {
+        let view = UIView()
+        view.backgroundColor = SiteTableViewControllerUX.HeaderBorderColor
+        return view
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension UITableViewController {
```

Comment:
Nit: I would make this private since it doesn't make sense to expose this publicly.